Add Unified Clean-up Process for Inactive Onboarding Trackers and Repositories (#201)
camaraproject/Governance: documentation/API-Onboarding-and-Lifecycle.md

### Review Cadence and Governance Flow
- Reviews are triggered **twice per year**, after each **meta-release cycle**.  
- The **API Backlog Working Group** conducts the review and notifies the responsible codeowners and maintainers via GitHub comment.  
- Contributors have a **4-week window** to provide updates, demonstrate progress, or transfer ownership to another CAMARA member.  
- **TSC involvement** is required before any archival:  
  - **Phase A–B:** TSC is informed and may confirm archival.  
  - **Phase C–D:** TSC must formally approve archival.  
- Reactivation is always possible through a **new API Proposal referencing the archived asset**.

### Archival Criteria per Phase

| **Phase** | **Description** | **Typical Triggers** | **Outcome** |
|------------|-----------------|----------------------|--------------|
| **A – Failed Onboarding** | Tracker created but no valid repository established. | No CODEOWNER, no EasyCLA, checklist empty, repository not created (after 4–6 weeks). | Backlog WG may reassign tracker; if not, TSC may confirm closure and archival. |

> | **B – Repository Without Artifacts** | Repository exists but contains no meaningful artifacts. | ≥ 3 of: no commits, no README/YAML, no validation, no operator engagement, not proposed for meta-release (after 6 months). | WG may seek new owner; if none, TSC informed before archival. |

Reviewer: How is "no operator engagement" measured? We should either define it clearly (e.g., "no operator reported in MAINTAINERS.md or APIbacklog.md") or remove if it's redundant with other criteria.

The ""≥ 3 of:" threshold is mechanistic but potentially inflexible, how about: typically ≥ 3 of the following indicators"

| **C – Repository With Partial Progress** | Some artifacts exist but no validation or adoption. | ≥ 3 of: YAML uploaded but no release, no validation, no WG activity, codeowner inactive, no commits for 6+ months. | Formal review by TSC; possible reassignment or archival. |
| **D – Repository With Reviewed Release (RC or 0.1.0+)** | Release exists but activity has stalled. | No activity post-release, maintainers left, no validation, or explicit request to archive. | Case-by-case TSC decision: archive, transfer, or defer. |

### Archival Actions

| **Action** | **Description** |
|-------------|----------------|

> | **Repository archived** | Repository set to read-only in GitHub and marked as *archived*. |

Reviewer: That's one and the same: 

```suggestion
| **Repository archived** | Repository archived on GitHub (becomes read-only with archived status badge). |
```
